fee_allocator/accounting/overrides.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod, ABCMeta
+from decimal import Decimal
+from typing import Dict, Type, TYPE_CHECKING
+from fee_allocator.constants import POOL_OVERRIDES_URL
+import requests
+
+if TYPE_CHECKING:
+    from fee_allocator.accounting.core_pools import PoolFee
+
+overrides_data = requests.get(POOL_OVERRIDES_URL).json()
+
+
+class PoolFeeOverrideMeta(ABCMeta):
```

Comment:
All these overrides are quite complex.  I think it'd be good to spend some time foucsing on docuemntation here.  What do all these classes/functions do?  Why are they needed/how do they relate to other things?  Are they setting defaults, what are the defaults/where can they be found?

Reply:
this was implemented when it wasnt clear how much fine tuned control we wanted over the override, but ya agree we can simplify this a lot, can probably just. be a dict